segbot_arm_tasks/src/object_to_office_task.cpp

```diff
@@ -0,0 +1,341 @@
+#include <ros/ros.h>
+#include <ros/package.h>
+
+#include <signal.h>
+
+#include <sensor_msgs/JointState.h>
+
+//srv for talking to table_object_detection_node.cpp
+#include "segbot_arm_perception/TabletopPerception.h"
+
+//action for grasping
+#include "segbot_arm_manipulation/TabletopGraspAction.h"
+#include "segbot_arm_manipulation/TabletopApproachAction.h"
+
+#include <segbot_arm_manipulation/arm_utils.h>
+#include <segbot_arm_manipulation/arm_positions_db.h>
+
+#include <segbot_arm_perception/segbot_arm_perception.h>
+
+
+#include "bwi_kr_execution/ExecutePlanAction.h"
+
+#include <move_base_msgs/MoveBaseAction.h>
+
+
+#include <moveit_utils/MicoNavSafety.h>
+
+#define NUM_JOINTS 8 //6+2 for the arm
+
+//mico joint state safe
+//-2.3321322971114142, -1.6372086401627464, -0.28393691436045176, -2.164605083475533, 0.7496982226688764, 4.682638807847723
+
+/* tool pose side
```

Comment:
If we need to save the joint positions of different arm configurations, we have the tools (in movit_utils) already to save and load bag files that contain arm joints state messages. We should use the tool rather than this printout